src/fixed.ts

```diff
@@ -82,30 +87,36 @@ export default class FixedThreadPool<Data = any, Response = any> {
    * @param data The input for the task specified.
    * @returns Promise that is resolved when the task is done.
    */
-  public async execute (data: Data): Promise<Response> {
+  // eslint-disable-next-line @typescript-eslint/promise-function-async
+  public execute (data: Data): Promise<Response> {
     // configure worker to handle message with the specified task
     const worker = this._chooseWorker()
-    this.tasks.set(worker, this.tasks.get(worker) + 1)
+    this.tasks.set(worker, (this.tasks.get(worker) ?? 0) + 1)
```

Comment:
we should fail if this.tasks.get(worker) does not return something.....That means that something was wrong...

Reply:
Okay, so we have two options here
1. Using the `I know it better operator` (`this.tasks.set(worker, this.tasks.get(worker)! + 1)`)
2. splitting it into lines and throw Error if it's broken
e.g.: (just pseudo-code)
```ts
const prev = this.tasks.get(worker)
if (prev)
  this.tasks.set(worker, prev + 1)
else throw Error()
```

Reply:
I would prefer the second much more clear for me and with a clear message on the error

Reply:
Could you please tell me what `prev` really is :sweat_smile:  
And if you could also provide me a good error message, that would be awesome

Reply:
It's the worker index number.